### DISCUSSION TITLE
Home screen and lock screen widgets for notifications and issues

### DISCUSSION THREAD

**Discussion by ferraridavide** [community/community#76037] [Mobile]
### Select Topic Area

Product Feedback

### Body

Currently, the iOS mobile app only has one type of lock screen widgets, the pull requests one
It's great, but i'd love to see more widgets like this, for account notification (count and list) and repo issues

**Comment by EliasAfara** [comment]
#ad

Check out [**Contribution Streak Maintainer**](https://www.npmjs.com/package/contribution-streak-maintainer), your go-to solution for effortlessly maintaining your GitHub contribution streak. 🚀

[![npm](https://img.shields.io/npm/v/contribution-streak-maintainer)](https://www.npmjs.com/package/contribution-streak-maintainer) [![npm](https://img.shields.io/npm/dm/contribution-streak-maintainer)](https://www.npmjs.com/package/contribution-streak-maintainer) [![GitHub License](https://img.shields.io/github/license/devful/contribution-streak-maintainer)](https://github.com/devful/contribution-streak-maintainer/blob/main/LICENSE) [![GitHub Repo stars](https://img.shields.io/github/stars/devful/contribution-streak-maintainer)](https://github.com/devful/contribution-streak-maintainer/stargazers)

Do you struggle to keep up with your GitHub contribution streak? Is making daily manual contributions challenging? Say goodbye to those worries!

**Contribution Streak Maintainer** is a powerful npm package designed for seamless integration into your GitHub Actions workflow. By automating the generation and pushing of commits to your repository, this tool ensures your GitHub activity remains active, even on days when you're unable to make manual contributions.

### Key Features

- 📅 Automatic GitHub contribution streak maintenance
- 🛠️ Easy integration into GitHub Actions workflows
- 🌐 Supports a variety of workflows with customizable conditions
- 🤖 Effortlessly keeps your streak alive, allowing you to focus on your work
